src/main/java/org/opentripplanner/standalone/config/RoutingRequestMapper.java

```diff
@@ -258,6 +260,20 @@ public static RouteRequest mapRoutingRequest(NodeAdapter c) {
 
     preferences.system().setDataOverlay(DataOverlayParametersMapper.map(c.path("dataOverlay")));
 
+    preferences
+      .street()
+      .setDrivingDirection(
+        mapDrivingDirection(c.path("drivingDirection"), preferences.street().drivingDirection())
+      );
```

Comment:
This bypasses the configuration mapper responsibilities, error handling and printing nice help when the config loading fails. We also make an implicit dependency to the configuration from the enum (configName). A better approach is to provide a mapping function to the `NodeAdampter.asEnum`. I can refactor it.